io/plugins/eu.esdihumboldt.hale.io.shp/src/eu/esdihumboldt/hale/io/shp/writer/ShapefileInstanceWriter.java

```diff
@@ -472,6 +478,8 @@ private Map<String, Map<String, ShapefileDataStore>> createSchema(URI location,
 	 * - filename_geometryType.shp if multiple geometries.<br>
 	 * - filename.shp single schema and geom.
 	 * 
+	 * Create the CPG file starting from the Shapefile names
```

Comment:
Missed moving this comment to the new method `writeCodePageFile` during refactoring?